cmd/sls/main.go

```diff
@@ -239,6 +251,45 @@ func main() {
 		Handler: router,
 	}
 
+	if cacheLayerEnable {
+		log.Printf("INFO: Enabling cache layer with capacity %d and TTL of %d seconds\n", cacheCapacity, cacheTTLSeconds)
+
+		// Setup Caching
+		memcached, err := memory.NewAdapter(
+			memory.AdapterWithAlgorithm(memory.LRU),
+			memory.AdapterWithCapacity(cacheCapacity),
+		)
+		if err != nil {
+			log.Fatalf("ERROR: Failed to setup memory cache adapter. Error: %s\n", err.Error())
+		}
+
+		cacheClient, err := cache.NewClient(
+			cache.ClientWithMethods([]string{http.MethodGet}),
+			cache.ClientWithAdapter(memcached),
+			cache.ClientWithTTL(time.Second*time.Duration(cacheTTLSeconds)),
+			// cache.ClientWithRefreshKey("opn"), // TODO
+		)
+		if err != nil {
+			log.Fatalf("ERROR: Failed to setup cache client. Error: %s\n", err.Error())
+		}
+
+		// Apply caching middleware, but exclude it from liveness readiness
+		router.Use(func(next http.Handler) http.Handler {
+			return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+				if r.URL.Path == "/v1/readiness" || r.URL.Path == "/v1/liveness" {
```

Comment:
Should /v1/health be included here too? I don't know if it should.
Also, the const's could be used: API_READINESS and API_LIVENESS